### PR TITLE
Read tasks from database

### PR DESCRIPTION
**Pull request by swillner**

### CODE REVIEW

**Comment by danielemden** [comment]
@swillner please review

File: main.go [hunk 1]
```diff
@@ -176,39 +178,52 @@ func main() {
...
+				filterMap[k] = true
+			}
+			fmt.Printf("%s: %s\n", k, v)
+		}
+
+		fmt.Println(filterMap)
+
+		// Find tasks in mongodb
+		findOptions := options.Find().SetProjection(bson.M{"_id": 0})
+		cur, err := tasksCollection.Find(r.Context(), filterMap, findOptions)
```
Comment (swillner) [inline main.go:199]: Do you need the `findOptions`?

File: main.go [hunk 2]
```diff
@@ -176,39 +178,52 @@ func main() {
...
 
-		w.Header().Set("Content-Type", "application/json")
+		// Load all from result
+		var results []bson.M
+		if err = cur.All(r.Context(), &results); err != nil {
+			log.Fatal(err)
+		}
+
+		// Convert to bytes
+		bytes, err := json.Marshal(results)
```
Comment (swillner) [inline main.go:213]: Is the way via `json` really necessary? `bson` should be able to unmarshal to json by itself...

File: main.go (hunk 1 above)
Comment (danielemden) [reply main.go:199]: Yes, we do not want to return the mongodb _id

Comment (swillner) [reply main.go:199]: ah, i see

**Pull request merged**